packages/kit/src/vite/index.js

```diff
@@ -320,12 +320,32 @@ function kit() {
 			process.env.SVELTEKIT_SERVER_BUILD_COMPLETED = 'true';
 			log.info('Prerendering');
 
-			prerendered = await prerender({
-				config: svelte_config.kit,
-				client_out_dir: vite_config.build.outDir,
-				manifest_path,
-				log
-			});
+			const results_path = `${svelte_config.kit.outDir}/generated/prerendered.json`;
+
+			// do prerendering in a subprocess so any dangling stuff gets killed upon completion
+			const kit_dir = createRequire(import.meta.url)
+				.resolve('@sveltejs/kit/package.json')
+				.slice(0, -'package.json'.length);
+			const { status } = spawnSync(
+				'node',
+				[
+					kit_dir + 'dist/prerender.js',
```

Comment:
would love it if we didn't lose the ability to test the `src` code — having to build `packages/kit` to test out simple changes is a pain

Reply:
https://github.com/sveltejs/kit/pull/5678#discussion_r927979693 it could work if you get the current file path (__dirname) and relative from that (checking for dist vs src in case the relative path is different)

Reply:
done